models/ticket_history/int_zendesk__field_history_pivot.sql

```diff
@@ -1,4 +1,4 @@
--- depends_on: {{ source('zendesk', 'ticket_field_history') }}
+-- depends_on: {{ ref('stg_zendesk__ticket_field_history') }}
```

Comment:
Same comment about why this is no longer pointing to the source? Is this because the source is deactivated if the union schema is being used?

Reply:
yep

Reply:
should we call out that folks may not be able to compile before running on a new schema?

Reply:
i'll add the `source` check for folks not using union_data